### PR TITLE
attempt to make test less flakey

### PR DESCRIPTION
**Pull request by maxtaco**
Give windows more time to be slow....

### CODE REVIEW

**Comment by maxtaco** [comment]
r? @patrickxb 

cc: @gabriel  who seems to hit it a lot

File: go/kex2/transport_test.go [hunk 1]
```diff
@@ -462,7 +462,7 @@ func TestErrAgain(t *testing.T) {
 
 func TestPollLoopSuccess(t *testing.T) {
 
-	wait := time.Duration(50) * time.Millisecond
+	wait := time.Duration(100) * time.Millisecond
```
Comment (patrickxb) [inline go/kex2/transport_test.go]: is `wait/32` on line below this still good?

Comment (maxtaco) [inline go/kex2/transport_test.go]: i think that's the repoll interval, so i think it's unrelated, yes...

gee, i wish i could redo all of this stuff with a virtualized clock, but ... :(

**Comment by gabriel** [comment]
In go, you can do `wait := 100 * time.Millisecond`

Also the CI machines can be really really slow. This might not fix it.

**Comment by maxtaco** [comment]
agreed! it's a workaround.....

**Pull request merged**